packages/uniforms-bridge-simple-schema-2/src/SimpleSchema2Bridge.ts

```diff
@@ -109,6 +117,12 @@ export default class SimpleSchema2Bridge extends Bridge {
     return undefined;
   }
 
+  private allowedValueToOption = <T>(value: T): Option<T> => ({
+    key: String(value),
+    label: String(value),
+    value,
+  });
```

Comment:
1. As above, there's no need to provide `key` and `label`.
2. There's no point in making it a method - either inline it entirely, or make a helper outside of the class.